metals/src/main/scala/scala/meta/internal/metals/scalacli/ScalaCli.scala

```diff
@@ -364,4 +364,34 @@ object ScalaCli {
 
   val name = "ScalaCli"
 
+  private def escape(s: String): String =
```

Comment:
You might use ujson for writing the Json and there will not be any need for escaping. Just a small suggestion, but I guess it was already like that before.